src/MessageSource.php

```diff
@@ -35,6 +35,11 @@ public function getMessage(string $id, string $category, string $locale, array $
         return dngettext($category, $id, $id, $n);
     }
 
+    public function getMessages(string $category, string $locale): array
+    {
+        throw new \RuntimeException('Function not allowed for this package.');
```

Comment:
I think we need to implement that by using Yii 2 implementations:

- https://github.com/yiisoft/yii2/blob/master/framework/i18n/GettextMoFile.php
- https://github.com/yiisoft/yii2/blob/master/framework/i18n/GettextPoFile.php